packages/rps/src/utils/rps-channel-client.ts

```diff
@@ -1,21 +1,15 @@
 import {AppData, ChannelState, encodeAppData, decodeAppData} from '../core';
-import {
-  ChannelClient,
-  ChannelResult,
-  Message,
-  ChannelClientInterface,
-} from '@statechannels/channel-client';
+import {ChannelResult, Message, ChannelClientInterface} from '@statechannels/channel-client';
 import {RPS_ADDRESS} from '../constants';
 import {bigNumberify} from 'ethers/utils';
 
 // This class wraps the channel client converting the request/response formats to those used in the app
 
 export class RPSChannelClient {
-  channelClient: ChannelClientInterface;
+  constructor(private readonly channelClient: ChannelClientInterface) {}
 
   async enable() {
```

Comment:
Why not remove this method?

Reply:
It's not obvious from the syntax, but this constructor is going to set `this.channelClient` to what you pass in to it. So it is the same as writing {`this.channelClient = channelClient`} in the body. https://www.typescriptlang.org/docs/handbook/classes.html#parameter-properties

Reply:
(and we need `this.channelClient` to be available to most of the class methods).

Reply:
Oh hah no I meant line 11 with the function implementation of `enable` being empty. I get what's happening on the line you explained :)